airbyte-integrations/connectors/source-file/setup.py

```diff
@@ -34,6 +34,12 @@
     "paramiko==2.7.2",
     "s3fs==0.5.2",
     "smart-open[all]==4.1.2",
+    "lxml==4.6.2",
+    "html5lib==1.1",
+    "beautifulsoup4==4.9.3",
```

Comment:
are all of these libs required? html/BS/pyarrow?

Reply:
Yes. This libraries required for read different file formats.